src/Acuminator/Acuminator.Vsix/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="Acumatica.Acuminator.778b19d1-1d5e-4fcb-8edb-eb103feeab7c" Version="1.6.0" Language="en-US" Publisher="Acumatica" />
+    <Identity Id="Acumatica.Acuminator.778b19d1-1d5e-4fcb-8edb-eb103feeab7c" Version="1.7.0" Language="en-US" Publisher="Acumatica" />
```

Comment:
Do you intentionally incremented a version number here?

Reply:
@metallmonster no, it was probably changed by git version

Reply:
Reverted this change